torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1483,7 +1485,7 @@ void InitXlaModuleBindings(py::module m) {
                       static_cast<XlaDeviceType>(xtensor->GetDevice().type())));
 
     at::Tensor cpu_tensor;
-    if (xla::sys_util::GetEnvBool("XLA_USE_SPMD", false) &&
+    if (ShardingUtil::UseVirtualDevice() &&
```

Comment:
Do we still need this given the XLA_CHECK above?

Reply:
Ah good catch! This can be removed